Support Request Headers (#156)
marcofranssen/terraform-provider-curl: docs/data-sources/request.md

### Optional

- `data` (String) The data sent in the request.

> - `headers` (Map of String) Key value headers to send with the request.

Reviewer: ```suggestion
- `headers` (Map of String) Headers sent in the request.
```
